ui/app/templates/components/config-pki-ca.hbs

```diff
@@ -9,7 +9,32 @@
   </h2>
   {{#if (or model.certificate model.csr)}}
     {{#each model.attrs as |attr|}}
-      {{info-table-row data-test-table-row label=(capitalize (or attr.options.label (humanize (dasherize attr.name)))) value=(get model attr.name)}}
+      {{#if (or 
+        (eq attr.name "csr")
```

Comment:
Is there something we could grab from the `attr.options` instead to indicate it should be masked? Instead of listing them out explicitly?

Reply:
Great idea.  Fixed.